viewer/packages/data-providers/src/image-360-data-providers/descriptor-providers/Cdf360CombinedDescriptorProvider.ts

```diff
@@ -25,17 +27,26 @@ export class Cdf360CombinedDescriptorProvider
     );
   }
 
-  constructor(fdmProvider: Cdf360DataModelsDescriptorProvider, eventProvider: Cdf360EventDescriptorProvider) {
+  constructor(
+    fdmProvider: Cdf360DataModelsDescriptorProvider,
+    eventProvider: Cdf360EventDescriptorProvider,
+    cdmProvider: Cdf360CdmDescriptorProvider
+  ) {
     this._fdmProvider = fdmProvider;
     this._eventProvider = eventProvider;
+    this._cdmProvider = cdmProvider;
   }
 
   public get360ImageDescriptors(
     metadataFilter: Metadata | Image360DataModelIdentifier,
     preMultipliedRotation: boolean
   ): Promise<Historical360ImageSet[]> {
     if (Cdf360CombinedDescriptorProvider.isFdmIdentifier(metadataFilter)) {
-      return this._fdmProvider.get360ImageDescriptors(metadataFilter, preMultipliedRotation);
+      if (metadataFilter.source === 'dm') {
+        return this._fdmProvider.get360ImageDescriptors(metadataFilter, preMultipliedRotation);
+      } else {
+        return this._cdmProvider.get360ImageDescriptors(metadataFilter, preMultipliedRotation);
+      }
```

Comment:
Shouldn't the default behavior (if `metadataFilter.source` is undefined)  be the old DM? Otherwise, we're breaking the API.

Reply:
Good catch, fixed in [b24ff79](https://github.com/cognitedata/reveal/pull/4894/commits/b24ff79808b01684a7688d8fc7e2782ba0bb5843)